b2b-sample/libs/business-admin-app/data-access/data-access-controller/src/lib/controller/controllerSwitchOrg/controllerDecodeSwitchOrg.ts

```diff
@@ -28,12 +28,12 @@ function getOrgId(token: JWT): string {
         if (token.user.user_organization) {
 
             return token.user.user_organization;
+        } else if (token.user.org_id) {
+
+            return token.user.org_id;
```

Comment:
Fix for the B2C user login issue. In B2C login cases, the user_organization claim won't be in the id token. The org_id claim of the token will represent the user_organization in such cases.